utils/python/tests/scripts_regression_tests

```diff
@@ -722,7 +722,7 @@ class FullSystemTest(TestCreateTestCommon):
                              msg="COMMAND SHOULD HAVE WORKED\wait_for_tests output:\n%s\n\nerrput:\n%s\n\ncode: %d" % (output, errput, stat))
 
 
-        stat, output, errput = run_cmd("%s/cs_status.%s" % (self._testroot, self._baseline_name), ok_to_fail=True)
```

Comment:
Hmmm, I wonder how this passed for me? Looks like old version was broken. Sorry if I pushed broken code.